install/kubernetes/quick-install.yaml

```diff
@@ -164,6 +164,7 @@ data:
   enable-xt-socket-fallback: "true"
   install-iptables-rules: "true"
   auto-direct-node-routes: "false"
+  enable-ipv4-fragment-tracking: "true"
```

Comment:
That file was generated by `make`. I think those are all the default values.

Reply:
I think if you want it to be omitted here then you need to change the configmap to have `{{ -if ... }}` logic wrapped around the option. (I don't have any opinion that this would help or hinder users)

Reply:
Yep, I followed the pattern of other options. I kinda makes sense to me that we'd want to list all options here. It makes it easier for the user to know whether the `quick-install.yml` will enable e.g., fragment tracking.